Units/parser-cxx.r/signature.cpp.d/input.cpp

```diff
@@ -2,7 +2,8 @@
 
 void foo (int a, char b) {}
 
-int bar (a, b) int a; char b; {}
+/* note that K&R style declarations are not valid in C++ */
+//int bar (a, b) int a; char b; {}
```

Comment:
Added a test for it.
Later I will look into extracting the parameters from K&R style function definitions too.